cocos/2d/CCNode.h

```diff
@@ -1862,8 +1866,13 @@ class CC_DLL Node : public Ref
 #if CC_USE_PHYSICS
     PhysicsBody* _physicsBody;
 public:
-    void setPhysicsBody(Component* physicsBody) 
-    { 
+    void setPhysicsBody(PhysicsBody* physicsBody)
```

Comment:
But it break compatibility. And we don't want Node depends on Physics related codes if possible, that's why we use component.